src/addremove.js

```diff
@@ -0,0 +1,71 @@
+class Store {
```

Comment:
There is an issue with your Add and remove functionality. It's not working...Take a good look at your code to correct this. So that your project can work better